sml_small/utils/pandas_example.py

```diff
@@ -0,0 +1,86 @@
+"""
+Example code that uses the pandas wrapper with the Totals and Components method.
+
+For Copyright information, please see LICENCE.
+"""
+import pandas as pd
+
+# We import the wrapper function from the pandas_wrapper
+from sml_small.utils.pandas_wrapper import wrapper
+
+
+# Function below is used to read a CSV file from the given
+# filepath and return as a DataFrame
+def load_csv(filepath):
+    df = pd.read_csv(filepath)
+    return df
+
+
```

Comment:
A comment to call out that this is the Totals and Components example usage might be good (see comment below)